src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -603,7 +603,7 @@ void RDMAConnectedSocketImpl::notify()
   int ret;
 
   ret = write(notify_fd, &i, sizeof(i));
-  assert(ret = sizeof(i));
+  assert(ret == sizeof(i));
```

Comment:
@amitkumar50 
The return value of `write()` should be compared with the bytes written. So comparing with `sizeof(i)` makes sense to me. Another point is this change may lead to a signed-unsigned comparison warning too, if not type casted appropriately.